x/nexus/keeper/genesis_test.go

```diff
@@ -117,19 +127,21 @@ func TestExportGenesisInitGenesis(t *testing.T) {
 
 	expected.ChainStates = []types.ChainState{
 		{
-			Chain:     axelarnet.Axelarnet,
-			Assets:    []string{axelarnet.Axelarnet.NativeAsset},
-			Activated: true,
+			Chain:        axelarnet.Axelarnet,
+			Assets:       []exported.Asset{exported.NewAsset(axelarnet.Uaxl, sdk.NewInt(100000))},
+			Activated:    true,
+			NativeAssets: []string{axelarnet.Uaxl},
 		},
 		{
 			Chain:     evm.Ethereum,
-			Assets:    []string{evm.Ethereum.NativeAsset, axelarnet.Axelarnet.NativeAsset},
+			Assets:    []exported.Asset{exported.NewAsset(axelarnet.Uaxl, sdk.NewInt(1000000))},
```

Comment:
we don't need to set the native asset for the evm chain?

Reply:
We are not using any native assets from evm, so I removed it.